src/components/Picker/Picker.visual.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { Picker } from './';
+
+class PickerVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualTestContainer>
+        <Picker {...this.props} />
+      </VisualTestContainer>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
```

Comment:
We should pass a `label` for this test

Reply:
what is label is used for ? also, would you like it to be with storybook-snapper ?